cloudera/datadog_checks/cloudera/config.py

```diff
@@ -0,0 +1,18 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+def normalize_config_clusters_include(log, clusters_config):
```

Comment:
I think it might be helpful to include a comment here to note that the `Discovery` class requires everything to be a key value pair, so this function is needed to smoothly convert yaml to json.